js/index.js

```diff
@@ -1 +1,102 @@
 // Your code goes here
+// mouseover ---- Used on "Let's Go!" img
+// keydown ------ Used on body bg color
+// wheel -------- Used on nav
+// drag / drop --
+// load --------- Used on window
+// focus -------- Used on form
+// resize -------
+// scroll ------- Used on fixed icon - to animate
+// select ------- Used on textarea
+// dblclick ---- Used on pick
+
+// ---------- SELECTORS ----------
+let destinationSection = document.querySelector(".content-destination");
+let destinationImg = document.querySelector(".content-destination img");
+const home = document.querySelector(".home");
+
+// =========== MOUSEOVER ==========
+let imgContent = document.querySelectorAll(".content-section .img-content img");
+imgContent[0].addEventListener("mouseover", function(e) {
+  e.target.style.opacity = ".6";
+  e.target.style.transition = ".4s";
+});
+
+destinationImg.addEventListener(
+  "mouseover",
+  e => (e.target.style.cursor = "pointer")
+);
+
+// =========== KEYDOWN ==========
+let body = document.querySelector("body");
+body.addEventListener("keydown", function(e) {
+  e.target.style.backgroundColor = "#aaa";
+  e.target.style.transition = "5s";
+});
+
+// =========== WHEEL ==========
+let nav = document.querySelector(".main-navigation");
+window.addEventListener("wheel", function() {
+  nav.style.backgroundColor = "#17A2B8";
+  nav.style.color = "white";
+  nav.style.borderBottom = "2px dashed #1782B8";
+  nav.style.transition = "1s";
+});
+
+// =========== LOAD ==========
+let tl = new TimelineMax();
+let logo = document.querySelector(".logo-heading");
+let navLinks = document.querySelectorAll(".nav-link");
+window.addEventListener("load", function() {
+  // alert("Welcome to Fun Bus!");
+  TweenMax.from(logo, 1, { y: -100 });
+  tl.staggerFrom(navLinks, 1, { y: 35 }, 0.2);
+  tl.from(home, 1, { opacity: 0 }, 1);
+});
+
+// =========== FOCUS ==========
+let form = document.querySelector(".cta-form");
+form.addEventListener(
+  "focus",
+  function(e) {
+    e.target.style.backgroundColor = "#17A2B8";
+  },
+  true
+);
+
+// =========== SCROLL ==========
+let funBusIcon = document.querySelector("#bus-icon img");
+let html = document.querySelector("html");
+window.addEventListener("scroll", function() {
```

Comment:
it would be cool if this animation fired once the user had scrolled all the way to the bottom of the page ... that way the bus would look like it's moving on a road created by the footer dashed border ... I don't know how to to that but it should be possible. I think that should be possible with the `scrollY` property on the Window object.

Reply:
Yes! I was trying to figure out how to do that but I ran out of time researching. Will definitely update the van animation in the future.